pyproject.toml

```diff
@@ -27,24 +26,42 @@ packages = [
     { include = "humanloop", from = "src"}
 ]
 
+
 [project.urls]
 Repository = 'https://github.com/humanloop/humanloop-python'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 httpx = ">=0.21.2"
 httpx-sse = "0.4.0"
 pydantic = ">= 1.9.2"
 pydantic-core = "^2.18.2"
 typing_extensions = ">= 4.0.0"
+parse = "^1.20.2"
+opentelemetry-sdk = "^1.27.0"
+opentelemetry-api = "^1.27.0"
+opentelemetry-instrumentation-openai = "^0.33.3"
+opentelemetry-instrumentation-cohere = "^0.33.3"
+opentelemetry-instrumentation-anthropic = "^0.33.3"
+opentelemetry-instrumentation-groq = "^0.33.3"
+opentelemetry-instrumentation-replicate = "^0.33.3"
 
-[tool.poetry.dev-dependencies]
```

Comment:
The dev dependencies were under two separate keys in the TOML file, I unified them

Reply:
In the future, use `poetry add --group dev <DEPENDENCY>` since other approaches have been deprecated